src/js/core/widget/core/SectionChanger.js

```diff
@@ -502,18 +504,21 @@
 				 * Changes the currently active section element.
 				 * @method setActiveSection
 				 * @param {number} index
-				 * @param {number} duration For smooth scrolling,
+				 * @param {number} [duration=0] For smooth scrolling,
 				 * the duration parameter must be in milliseconds.
-				 * @param {number} [direct=false]
+				 * @param {number} [direct=false] Wheter section is set once directly (e.g. bezel)
+				 *  or with touch events
 				 * @member ns.widget.core.SectionChanger
 				 */
 				setActiveSection: function (index, duration, direct) {
 					var position = this.sectionPositions[index],
-						scrollbarDuration = duration,
+						scrollbarDuration = (duration = (duration || 0)),
```

Comment:
Understanding this expression requires too much effort for me.

I prefer simple (safe) solution like:
```
var position = this.sectionPositions[index],
      scrollbarDuroation,
      oldActiveIndex = this.activeIndex,
      newX = 0,
      newY = 0;

duration = duration || 0;
scrollbarDuration = duration;
...
```

Reply:
done